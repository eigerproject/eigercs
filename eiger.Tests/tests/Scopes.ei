let x = 10

if x ?= 10 then
	let y = 20
	emit(y)
end

emitln(x)

~ ---------------------- ~

x = 10

if x ?= 10 then
	x = 20
	emit(x)
end

emitln(x)

~ ---------------------- ~

x = 5

func example()
	let y = x + 2
	emitln(y)
end

example()

~ ---------------------- ~

x = 5

func secondExample()
	x = 10
	emit(x)
end

secondExample()
emitln(x)