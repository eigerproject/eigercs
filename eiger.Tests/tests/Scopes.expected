2010
2020
7
1010